src/Agent.Worker/Build/TfsVCSourceProvider.cs

```diff
@@ -131,7 +131,7 @@ public sealed class TfsVCSourceProvider : SourceProvider, ISourceProvider
                             if (definitionMapping.MappingType == DefinitionMappingType.Map)
                             {
                                 // Check the status.
-                                string localPath = ResolveMappingLocalPath(definitionMapping, sourcesDirectory);
+                                string localPath = definitionMapping.GetRootedLocalPath(sourcesDirectory);
```

Comment:
I moved ResolveMappingLocalPath onto the DefinitionMapping class itself. It really makes more sense there anyway, but the real motivation was that I need to be able to reference it from WorkspaceUtil class.